static/scripts/onboarding/onboarding.ts

```diff
@@ -547,22 +545,34 @@ const step2Handler = async () => {
 
     await erc20.approve(PERMIT2_ADDRESS, parseUnits(allowance.toString(), decimals));
     singleToggle("success", `Success`);
-  } catch (error) {
+  } catch (error: any) {
```

Comment:
This should be typed as an error correctly. We should not allow the use of `any`

Reply:
This is kind of an edge case because here `error` can be either of the `Error` type either of the `string` type

So if `error` has a `string` type then `console.log(error)` works fine because `error` is a `string` by default

If `error` has an `Error` type then `console.log(error)` works fine because under the hood `Error.toString()` is called which is the same as `error.message`

TLDR;

`error: any` is more concise than:
```
catch(e){
    result = e.message; // error under useUnknownInCatchVariables 
    if (typeof e === "string") {
        e.toUpperCase() // works, `e` narrowed to string
    } else if (e instanceof Error) {
        e.message // works, `e` narrowed to Error
    }
}
```

Reply:
> can be either of the Error type either of the string type

If it is a string, then we should be doing something like `if (typeof errorString == "string") { return new Error(errorString) }`

We should really not be using `any`

Reply:
https://github.com/ubiquity/pay.ubq.fi/issues/135